src/routes/rules/RulesView.vue

```diff
@@ -161,129 +154,99 @@
-          <li>The hand limit is reduced to 7 (from 8).</li>
+          <li>{{ `${t('rules.3PRule4')}` }}</li>
         </ul>
 
-        <h3>4 Player Cuttle: 2v2</h3>
+        <h3>{{ `${t('rules.4PTitle')}` }}</h3>
         <p class="mb-4">
-          4 player Cuttle is a 2v2 team game. Each player sits across from their partner. Your team wins if
-          either you or your teammate reach your respective goals, but players do not share cards on their
-          hand or field (points are counted separately and your Royals do not directly affect your partner).
+          {{ `${t('rules.4PText')}` }}
         </p>
         <p>
-          The rules are otherwise the same as 1v1 Cuttle, with play proceeding clockwise from the left of the
-          dealer, with the following other changes:
+          {{ `${t('rules.4PRules')}` }}
         </p>
         <ul>
           <li>
-            Each player is dealt 5 cards, play starts left of the dealer and proceeds clockwise from there.
+            {{ `${t('rules.4PRule1')}` }}
           </li>
           <li>
-            Shuffle two Jokers into the deck. Jokers count as Royals with the effect "Steal target Royal".
-            Similar to Jacks, they may transfer control of a Royal from any player to any other player.
+            {{ `${t('rules.4PRule2')}` }}
           </li>
-          <li>The hand limit is reduced to 7 (from 8).</li>
+          <li>{{ `${t('rules.4PRule3')}` }}</li>
           <li>
-            Jacks may be played to transfer control of any point card to any other player. You may therefore
-            "steal" points from yourself (or anyone else) in order to give them to your partner, potentially
-            winning on your turn with their points (their goal is still determined by the count of their own
-            Kings on the field).
+            {{ `${t('rules.4PRule4')}` }}
           </li>
           <li>
-            Nines can be played for an alternative, similar effect. Whenever an opponent plays a Royal, you
-            may immediately play a Nine in response to return that card to its owner's hand immediately,
-            preventing its effect from taking place. Doing so may be done out of turn and does not take your
-            turn, but does not prevent the target's owner from playing the card again on their next turn.
-            Alternatively, you may still play a Nine for its original one-off effect on your turn (taking your
-            turn to do so) to return a card to its controller's hand and prevent them from playing that card
-            on their next turn.
+            {{ `${t('rules.4PRule5')}` }}
           </li>
         </ul>
       </v-row>
 
       <!-- FAQ -->
       <v-row class="d-flex flex-column mb-4">
         <h1 class="gradient-text">
-          FAQ
+          {{ `${t('rules.faqTitle')}` }}
         </h1>
-        <h3>Can I play a two to counter a point card? What about a scuttle?</h3>
+        <h3>{{ `${t('rules.twoCounter')}` }}</h3>
```

Comment:
let's preface all the faq translations with `faq` similar to what you have for `faqTitle`: 
```suggestion
        <h3>{{ `${t('rules.faqTwoCounter')}` }}</h3>
```

Reply:
Same for all the others in the FAQ section

Reply:
Resolved in commit 84cb820